sdk/purestorageblock/Azure.ResourceManager.PureStorageBlock/tests/PureStorageBlockManagementTestBase.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.TestFramework;
+using NUnit.Framework;
+using System.Threading.Tasks;
+
+namespace Azure.ResourceManager.PureStorageBlock.Tests
+{
+    public class PureStorageBlockManagementTestBase : ManagementRecordedTestBase<PureStorageBlockManagementTestEnvironment>
```

Comment:
scenario tests should be added for operations, refer to https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/workloadssapvirtualinstance/Azure.ResourceManager.WorkloadsSapVirtualInstance/tests/Tests/SapVirtualInstanceCRUDTests.cs for example.
Test details can be found in https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md

Reply:
The scenario tests are recommended for beta version .NET SDK, but mandatory for GA versions.
This will not block this PR merge.